console/client/src/features/timeline/Timeline.tsx

```diff
@@ -42,6 +42,12 @@ export const Timeline = () => {
       abortController.abort()
     }
   }, [ client ])
+  
+  useEffect(() => {
+    if(!isOpen) {
+      setSelectedEntry(null)
+    }
+  }, [ isOpen ])
```

Comment:
What's this `useEffect` doing (so I can understand it a bit better)?

Reply:
deselect when you close the sidepanel cause some weird feedback states otherwise